src/nile/deployments.py

```diff
@@ -49,9 +49,15 @@ def register_class_hash(hash, network, alias):
         fp.write("\n")
 
 
-def exists(identifier, network):
-    """Return whether a deployment exists or not."""
-    foo = next(load(identifier, network), None)
+def exists(address_or_alias, network):
+    """
+    Return whether a deployment exists or not.
+
+    If address_or_alias is an int, address is assumed.
+
+    If address_or_alias is a str, alias is assumed.
+    """
+    foo = next(load(address_or_alias, network), None)
     return foo is not None
```

Comment:
```suggestion
    deployment = next(load(address_or_alias, network), None)
    return deployment is not None
```
Don't forget about `foo`!